README.md

```diff
@@ -34,3 +34,17 @@ cd prio
 make
 make test
 ```
+
+## Adding new Python dependencies
+
+To add new Python dependencies to the container, use `pip-tools` to manage the
+`requirements.txt`.
+
+```bash
+pip install pip-tools
+
+# add new requirement to requirements.in
+
+# generate a new requirements.txt
+pip-compile
```

Comment:
```suggestion
pip-compile --generate-hashes
```

Reply:
Unfortunately, the local repositories are unhashable and cause pip-tools to fail when using `--generate-hashes`.

```
FileNotFoundError: [Errno 2] No such file or directory: '/./processor'
```

Reply:
oh that's a downer :(

Maybe you could ask if there's a way to make this work on https://chat.mozilla.org/#/room/#python:mozilla.org ? I'm basically ok with landing as-is in any case, though it feels like the sort of thing that maybe we should try to get right now, while we're looking at it.

Reply:
This would probably work if the dependencies were added to the `requirements.in` without the actual prio/processor packages. I think this defeats the purpose of a top-level environment requirements though.

But I've had recent experience `pip-compile`ing a vcs package (e.g. git+https://github.com/...). Those are also unhashable -- there needs to be some sort of binary like an zip or wheel to hash. I could use [`hashin`](https://pypi.org/project/hashin/) to generate the hashes directly for built wheels/eggs, but in this repository, `prio` and `prio-processor` aren't built until docker build time. 

Here's a blog post that I think does a good job at explaining some of the issues here, including the unsafe one below: https://lil.law.harvard.edu/blog/2019/05/20/improving-pip-compile-generate-hashes/